packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -386,12 +383,6 @@ function executeDispatchesInOrder(event) {
   event._dispatchInstances = null;
 }
 
-/**
- * Internal queue of events that have accumulated their dispatches and are
- * waiting to have their dispatches executed.
- */
-let eventQueue: ?(Array<ReactSyntheticEvent> | ReactSyntheticEvent) = null;
```

Comment:
This and a bunch of other stuff was unnecessary because we only dispatch one event at a time in this file.